src/common/models/Projection.ts

```diff
@@ -491,6 +514,44 @@ export class Projection {
     );
   }
 
+  private calcCaseDensityCases(
+    actualTimeseries: Array<CANActualsTimeseriesRow | null>,
+  ): Array<number | null> {
+    return actualTimeseries.map(
+      (row: CANActualsTimeseriesRow | null, i: number) => {
```

Comment:
Since you're not actually using anything from actualTimeseries, it's a bit weird to map over it.

Perhaps the simplest would just be:

```
for(let i = 0; i < this.dates.length; i++) {
...
}
```

Since dates represents all the dates in our timeseries.